recipes/sentry-native/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SentryNativeConan(ConanFile):
+    name = "sentry-native"
+    description = "Sentry SDK for C, C++ and native applications."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/getsentry/sentry-native"
+    license = "MIT"
+    topics = ("conan", "breakpad", "crashpad",
+              "error-reporting", "crash-reporting")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```

Reply:
See https://github.com/madebr/sentry-native/blob/sentry_conan/conan/conanfile.py for a recipe meant for sentry developers